### ISSUE TITLE
Workaround firefox ubuntu bug

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/metamask-extension#4791]
Firefox on ubuntu can't show password boxes. We should:

- Detect this system on popup, and open full screen mode.
- In old UI, add `align-items: center;` to `#app-content`.

**Comment by bdresser** [comment]
relates to (and might fix) #4502 #3963

**Comment by estebanmino** [comment]
I tried another strategy to solve this and I came to a Ubuntu 18.04 specific issue related to Ibus (https://github.com/ibus/ibus/). The default keyboard input method on Ubuntu 18.04 is Ibus and it seems that this version of Ubuntu has a bug with Gnome (here is the proposed patch https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/140/diffs?commit_id=94cca97ad599f1da9bc52d8ff5cb28df681a3538).

The way that I fixed this, MetaMask working for all Firefox versions on Ubuntu, was changing the keyboard input method system, from Ibus to XIM. This change could lead to problems with Japanese or Chinese support but it fixes the issues mentioned before.
